app/src/main/java/org/mozilla/fenix/settings/DeleteBrowsingDataItem.kt

```diff
@@ -61,4 +66,10 @@ class DeleteBrowsingDataItem @JvmOverloads constructor(
             subtitle.text = resources.getString(subtitleId)
         }
     }
+
+    override fun setEnabled(enabled: Boolean) {
+        super.setEnabled(enabled)
+        alpha = if (enabled) ENABLED_ALPHA else DISABLED_ALPHA
+        checkbox.isEnabled = enabled
```

Comment:
Is this line needed? As far as I can tell, I can't use the checkbox if `DeleteBrowsingDataItem` is disabled.

Reply:
Yeah, got this wrong, re-pushing.

Reply:
Pushed change to include this.